MPCAutofill/cardpicker/templates/cardpicker/legal.html

```diff
@@ -41,8 +75,7 @@ <h2>Privacy Policy</h2>
     <a href="https://tools.google.com/dlpage/gaoptout" target="_blank">here</a>.
     </p>
 
-    If you have any questions about our Privacy Policy or would like to otherwise get in touch, you can contact us at:
-    <a href="mailto:mpcfillsite@gmail.com">mpcfillsite@gmail.com</a>
+    If you have any questions about our Privacy Policy or would like to to request a takedown of artistic material you own that may have been misattributed by a drive owner, feel free to email us at: mpcfillsite@gmail.com

```

Comment:
having this as `<a href="mailto:mpcfillsite@gmail.com">mpcfillsite@gmail.com</a>` means you can click the link to open in your email client

Reply:
Added! I also went ahead and made this line use the email ENV and set a default value as well, not sure how I didn't think of that before lol